teamcity/templates/readme/readme.md.jinja2

```diff
@@ -29,6 +29,5 @@
 | Change ID | Author | Description |
 | --------- | ------ | ----------- |
 {%- for commit in commits %}
-| {{ commit.hexsha | truncate(7, true, '') }} | {{ commit.author }} | {{ commit.message | trim | replace('\n', '<br>') }} |
-{%- endfor -%}
-
+| {{ commit.hexsha | truncate(7, true, '') }} | {{ commit.author }} | {{ commit.message | trim | replace('\r', '') | replace('\n\n', '<br>') | replace('\n', '<br>') | replace('*', '&bull;')}}|
```

Comment:
Why was an explicit replace for `\r` added here? Commit messages should never contain carriage returns by the standard of git.

Also, a space is missing before the last `}}`